### PR TITLE
Add support for specifying the semver-range

### LINKED ISSUE #2: Add a way to pass in tag/version
Need to be able to do the equivalent 

``await latestVersion('npm', {version: 'latest-5'})``

### PR DESCRIPTION
**Pull request by LitoMore**
### Changes

- Add support for specifying the [dist-tag](https://docs.npmjs.com/adding-dist-tags-to-packages) by passing the flag `--version`.
- Upgrade devDependencies to the latest version.
- Requrie Node.js 8.
- Add `.npmrc` file to disable npm lockfile.

Fixes #2

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -7,14 +7,30 @@ const cli = meow(`
 	Usage
 	  $ latest-version <package-name>
 
+	Options
+	  -v         Ouput the version of this package
+	  --version  Specify which dist-tag to use to find the latest version
```
Comment (sindresorhus) [inline cli.js:12]: The description here could be better. It's not just the dist-tag, but it could also be a semver-range.

**Comment by sindresorhus** [comment]
Could we maybe use a better flag name than `--version`? It could be for either a dist-tag or semver range.

File: cli.js (hunk 1 above)
Comment (LitoMore) [reply cli.js:12]: Good point.

**Comment by LitoMore** [comment]
> Could we maybe use a better flag name than --version? It could be for either a dist-tag or semver range.

My original intention was to make them all use the same option names. The `version` is defined [here](https://github.com/sindresorhus/package-json#options).

**Comment by sindresorhus** [comment]
There's a difference though. `package-json` can get the exact version. That makes no sense here. So we should not be bound to the same naming.

**Comment by sindresorhus** [comment]
Bump :)

**Comment by LitoMore** [comment]
What about the name `--scope`? I will document the `scope` that can be for either a dist-tag or semver range.

**Comment by sindresorhus** [comment]
I think that would be confusing as "**scoped** packages" is a different thing

**Comment by sindresorhus** [comment]
Let's go with `--range`.

**Comment by sindresorhus** [comment]
Closing for lack of response.

https://github.com/sindresorhus/latest-version-cli/issues/2#issuecomment-701060551

**Pull request closed without merging**